lib/utils/utils.go

```diff
@@ -439,6 +441,10 @@ func ReadHostUUID(dataDir string) (string, error) {
 func WriteHostUUID(dataDir string, id string) error {
 	err := ioutil.WriteFile(filepath.Join(dataDir, HostUUIDFile), []byte(id), os.ModeExclusive|0400)
```

Comment:
```suggestion
	err := os(filepath.Join(dataDir, HostUUIDFile), []byte(id), os.ModeExclusive|0400)
```

Unrelated, but while you're here..